lab_2_tokenize_by_bpe/main.py

```diff
@@ -69,6 +133,16 @@ def get_vocabulary(
     :param unknown_token: a token to signify an unknown token
     :return: dictionary in the form of <token: identifier>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(unknown_token, str):
+        return None
+    unique_tokens = set()
+    for word in word_frequencies.keys():
+        unique_tokens.update(word)
+    sorted_tokens = sorted(unique_tokens, key=lambda uni_token: (-len(uni_token), uni_token))
```

Comment:
sets do not have particular order, to sorting a set is not a good idea

Reply:
maybe this way it's better